packages/react-native/cli.js

```diff
@@ -67,7 +67,7 @@ async function main() {
       // Ignore errors, since it's a nice to have warning
     }
   }
-  return cli.run();
+  return cli.run('react-native');
```

Comment:
More context: https://github.com/react-native-community/cli/pull/2170

Reply:
Let's use name from package.json that way each out of tree platform will work out of the box